modules/introduction/pages/whats-new.adoc

```diff
@@ -1,193 +1,108 @@
+[#whats-new-query-660]
+=== Query Service
 
-Window functions are used to compute an aggregate or cumulative value, based on a group of objects. The objects are not grouped into a single output object — each object remains separate in the query output.
+==== Flex Index
 
-For more information, see xref:n1ql:n1ql-language-reference/windowfun.adoc[Window Functions].
+Futher to the N1QL and Full-Text Search (FTS) integration introduced in release 6.5, release 6.6 introduces N1QL Flex Index, which enables applications to leverage Couchbase FTS index transparently in N1QL statements. 
 
-===== Common Table Expressions (CTE)
+Applications with ad hoc queries can now benefit from FTS index while using standard N1QL predicates. The USE INDEX (USE FTS) hint steers the Query Service to automatically choose an applicable FTS index that is best suited for the query. 
 
-Couchbase Common Table Expression is yet another ANSI CTE equivalent to enhance the manageability of complex queries where the developer can isolate the SQL statement into an ad hoc named result set, which can be referenced as a source table in the context of a larger query.
+To learn more about the Flex Index feature, see xref:n1ql:n1ql-language-reference/flex-indexes.adoc[Flex Indexes].
 
-You can use the WITH clause to create a common table expression. For more information, see xref:n1ql:n1ql-language-reference/with.adoc[WITH Clause].
+==== Index Advisor 
 
-===== Query and Search Integration
+“Index Advisor” is promoted as a supported feature, and is now available as an online tutorial. 
 
-Couchbase’s powerful full-text search capability can now be fully integrated with N1QL. This feature extends N1QL search functionality with fuzzy and non-exact text searches across multiple fields and any order. By enabling developers to use full-text search queries directly within a N1QL query, it eliminates the need to write complex code to process and combine the results from separate SQL and search queries.
 
-For more information, see xref:n1ql:n1ql-language-reference/searchfun.adoc[Search Functions].
+The Index Advisor eliminates the guesswork in index creation and recommends changes to existing indexes wherever applicable. By providing recommendations for a specific query, or a set of queries from the Couchbase server log, it helps you create optimum indexes for your queries. 
 
-===== Cost-Based Optimizer (Developer Preview)
+The Index Advisor also enables you to monitor performance and manage indexes to reflect the current usage pattern. 
 
-Couchbase delivers a cost-based optimizer, deriving access patterns based on statistical data for query tuning to the NoSQL world. This feature helps developers by eliminating the time spent on tweaking a query using a rule-based optimizer to choose the most optimal execution plan.
+Starting with this release, the Index Advisor is a supported feature and is available as an https://index-advisor.couchbase.com[online tutorial] too. 
```

Comment:
```suggestion
Starting with this release, the Index Advisor is a supported production feature and is available through graphical interfaces like the https://index-advisor.couchbase.com[online tutorial (new)^] and the xref:tools:query-workbench.adoc#index-advisor[Query Workbench]. It can also be invoked by using the xref:n1ql:n1ql-language-reference/advise.adoc[ADVISE Statement].
```